cypress/integration/search/globalSearch.js

```diff
@@ -0,0 +1,38 @@
+const GLOBAL_SEARCH_URL =
+    '/iframe.html?id=modules-search-stories-globalsearch--algolia-global-search';
+const AGOLIA_URL = 'https://6mt6rezjme-dsn.algolia.net/1/indexes/rainbow-books/query';
+const SEARCH_INPUT = 'input[type="search"]';
+
+const searchBooksAlgolia = () =>
+    cy
+        .intercept('POST', AGOLIA_URL, { fixture: 'globalSearchResult.json' })
+        .as('searchBooksAlgolia');
+
+describe('GlobalSearch', () => {
+    beforeEach(() => {
+        cy.visit(GLOBAL_SEARCH_URL);
+        searchBooksAlgolia();
+    });
+
+    it('should open the GlobaSearch', () => {
+        cy.get(SEARCH_INPUT).click();
+        cy.get('h1').contains('Type something to search').should('exist');
+    });
+
+    it('should close the GlobaSearch', () => {
+        cy.get(SEARCH_INPUT).click();
+        cy.get('button').click();
```

Comment:
this is the close button? If so, we should call the test `should close the GlobaSearch when clicking the close button`, also we should find the button in a more deterministic way, we can discuss it